data/net/minecraft/client/gui/GuiGraphics.mapping

```diff
@@ -7,409 +7,866 @@ CLASS net/minecraft/client/gui/GuiGraphics
 		ARG 2 maxX
+			COMMENT the x-coordinate of the end point.
 		ARG 3 y
+			COMMENT the y-coordinate of the line.
 		ARG 4 color
+			COMMENT the color of the line.
 	METHOD hLine (Lnet/minecraft/client/renderer/RenderType;IIII)V
+		COMMENT Draws a horizontal line from minX to maxX at the specified y-coordinate with the given color using the specified render type.
 		ARG 1 renderType
+			COMMENT the render type to use.
 		ARG 2 minX
+			COMMENT the x-coordinate of the start point.
 		ARG 3 maxX
+			COMMENT the x-coordinate of the end point.
 		ARG 4 y
+			COMMENT the y-coordinate of the line.
 		ARG 5 color
+			COMMENT the color of the line.
 	METHOD innerBlit (Lnet/minecraft/resources/ResourceLocation;IIIIIFFFF)V
+		COMMENT Performs the inner blit operation for rendering a texture with the specified coordinates and texture coordinates without color tinting.
 		ARG 1 atlasLocation
+			COMMENT the location of the texture atlas.
 		ARG 2 x1
+			COMMENT the x-coordinate of the first corner of the blit position.
 		ARG 3 x2
+			COMMENT the x-coordinate of the second corner of the blit position.
 		ARG 4 y1
+			COMMENT the y-coordinate of the first corner of the blit position.
 		ARG 5 y2
+			COMMENT the y-coordinate of the second corner of the blit position.
 		ARG 6 blitOffset
+			COMMENT the z-level offset for rendering order.
 		ARG 7 minU
+			COMMENT the minimum horizontal texture coordinate.
 		ARG 8 maxU
+			COMMENT the maximum horizontal texture coordinate.
 		ARG 9 minV
+			COMMENT the minimum vertical texture coordinate.
 		ARG 10 maxV
+			COMMENT the maximum vertical texture coordinate.
 	METHOD innerBlit (Lnet/minecraft/resources/ResourceLocation;IIIIIFFFFFFFF)V
+		COMMENT Performs the inner blit operation for rendering a texture with the specified coordinates, texture coordinates, and color tint.
 		ARG 1 atlasLocation
+			COMMENT the location of the texture atlas.
 		ARG 2 x1
+			COMMENT the x-coordinate of the first corner of the blit position.
 		ARG 3 x2
+			COMMENT the x-coordinate of the second corner of the blit position.
 		ARG 4 y1
+			COMMENT the y-coordinate of the first corner of the blit position.
 		ARG 5 y2
+			COMMENT the y-coordinate of the second corner of the blit position.
 		ARG 6 blitOffset
+			COMMENT the z-level offset for rendering order.
 		ARG 7 minU
+			COMMENT the minimum horizontal texture coordinate.
 		ARG 8 maxU
+			COMMENT the maximum horizontal texture coordinate.
 		ARG 9 minV
+			COMMENT the minimum vertical texture coordinate.
 		ARG 10 maxV
+			COMMENT the maximum vertical texture coordinate.
 		ARG 11 red
+			COMMENT the red component of the color tint.
 		ARG 12 green
+			COMMENT the green component of the color tint.
 		ARG 13 blue
+			COMMENT the blue component of the color tint.
 		ARG 14 alpha
+			COMMENT the alpha component of the color tint.
+	METHOD pose ()Lcom/mojang/blaze3d/vertex/PoseStack;
+		COMMENT {@return returns the PoseStack used for transformations and rendering.}
```

Comment:
The word "Returns" and the final period should not be used for the inline `{@return}` tag, as the tag itself will insert them.
```suggestion
		COMMENT {@return the PoseStack used for transformations and rendering}
```